docs/preview/quarantine/readme.md

```diff
@@ -0,0 +1,223 @@
+```
+
+## Quarantined Webhook Notification
+
+Once the image is pushed, you will receive a notification through webhooks.
+
+## Pull the quarantined image
+
+Once the image is quarantined, you will need user with the **AcrQuarantineReader** role. The presumption here is the Vulnerability Scanning solution is configured to use this account.
+
+`docker login ${REGISTRY_NAME} -u` **[quarantinedServicePrincpalUsr]**` -p `**[quarantinedServicePrincpalPwd]**
+
+Now the user can pull the quarantined image by digest
+
+```
+docker pull [registry].azurecr.io/helloworld@sha256:sha256:80f0d5cxxxxXxxXxxxxece0db56d11cdc624ad20da9fe62d7d
+  Pulling from helloworld
+	Digest: sha256:80f0d5cxxxxXxxXxxxxece0db56d11cdc624ad20da9fe62d7d
+	Status: Image is up to date for [registry].azurecr.io/helloworld@sha256:80f0d5cxxxxXxxXxxxxece0db56d11cdc624ad20da9fe62d7d
+```
+
+## Query Attributes of an Image (manifest)
+
+Querying ACR metadata via the REST API requires an OAuth Token.
+
+To query attributes via REST, use the following workflow:
+
+1. encode the username/password with the required permissions
+1. get an access token
+1. query acr metadata for the attributes of a given digest
+
+### Encode the username and password 
+  - using a tool like https://www.base64encode.org/
+    - Encode using the following format: **[username]**:**[password]**
+    - Copy the encoded value
+
+### Get an  access token for the user
+
+- Using a REST client, like Postman, query for the OAuth token:
+
+    REST format: `https://`**[login-url]**`/oauth2/token?service=`**[login-url]**`&scope=repository:`**[image]**`:pull`
+
+- Set the header for Authorization, setting the 'Basic' word followed by a space and the encoded usr:pwd value
+
+    |Header | Value |
+    |-------|-------|
+    | Authorization | Basic [base64 encoded usr:pwd] |
+    | Host | [login-url] |
+
+    example: 
+    ```
+    GET https://quarantinetest1.azurecr.io/oauth2/token?service=quarantinetest1.azurecr.io&scope=repository:helloworld:pull
+    ```
+
+
+### Query the metadata API
+
+- With an OAuth token, we can now query ACR metadata:
+
+    REST format: `https://`**[login-url]**`/acr/v1/`**[image]**`/_manifests/`**[digest]**
+
+- Set the header for Authorization, setting the OAuth token
+
+    |Header | Value |
+    |-------|-------|
+    | Authorization | Bearer [token] |
+    | Host | [login-url] |
+
+    example: 
+    ```
+    GET https://quarantinetest1.azurecr.io/acr/v1/mytest/_manifests/sha256:80f0d5cxxxxXxxXxxxxece0db56d11cdc624ad20da9fe62d7d
+    ```
+## Remove the Quarantine Flag
+
+Once a scan completes, a user with the **AcrQuarantineWriter** role can update the manifest attribute to removed the quarantined flag.
+
+### Encode the username and password 
+  - using a tool like https://www.base64encode.org/
+    - Encode using the following format: **[username]**:**[password]**
+    - Copy the encoded value
+
+### Get a Push access token for the user

```

Comment:
Would point to the getting access token section rather than describing how here. Basically we can add a section there instead of this doc. Want to keep all access token doc's together.

Reply:
Added a separate document on getting token using Basic Auth, and refer to it here.